docs/source/features/sheet_management/index.rst

```diff
@@ -37,24 +40,60 @@ necessary to get a dataset first:
  Create a Sheet
 ****************
 
-Create a sheet on the open draft:
+SDK supports managing sheets of the open draft. Thus, it is necessary to create or get an open
+draft:
 
 .. code:: python
 
-   from graviti import DataFrame
-
    draft = dataset.drafts.create(f"{DRAFT_TITLE}")
-   draft[f"{SHEET_NAME}"] = DataFrame(data=data, schema=schema)
-   draft[f"{SHEET_NAME}"] = DataFrame(schema=schema)
+
+Then users need to :ref:`features/sheet_management/dataframe:Initialize a DataFrame` like:
+
+.. code:: python
+
+   >>> from graviti import DataFrame
+   >>> data = [
+   ...    {"filename": "a.jpg"},
+   ...    {"filename": "b.jpg"},
+   ...    {"filename": "c.jpg"},
+   ... ]
+   >>> df = DataFrame(data)
+   >>> df
+      filename
+   0  a.jpg
+   1  b.jpg
+   2  c.jpg
+ 
+Or getting the dataframe from other sheets:
```

Comment:
Is `from other sheets` appropriate?  Should it be `getting a dataframe by copying other sheets`?